server/channels/api4/apitestlib.go

```diff
@@ -555,6 +555,24 @@ func (th *TestHelper) CreateLocalClient(socketPath string) *model.Client4 {
 	}
 }
 
+func (th *TestHelper) CreateConnectedWebSocketClient(t *testing.T) *model.WebSocketClient {
+	t.Helper()
+	wsClient, err := th.CreateWebSocketClient()
+	require.NoError(t, err)
+	require.NotNil(t, wsClient, "webSocketClient should not be nil")
+	wsClient.Listen()
```

Comment:
Should the helper also close the websocket client? 
```suggestion
	wsClient.Listen()
	t.Cleanup(wsClient.Close)
```

Reply:
At the moment, it's closed by the caller, but leveraging `t.Cleanup` looks good as well.